src/modinv32.h

```diff
@@ -35,4 +35,8 @@ static void secp256k1_modinv32_var(secp256k1_modinv32_signed30 *x, const secp256
 /* Same as secp256k1_modinv32_var, but constant time in x (not in the modulus). */
 static void secp256k1_modinv32(secp256k1_modinv32_signed30 *x, const secp256k1_modinv32_modinfo *modinfo);
 
+/* Compute the Jacobi symbol for (x | modinfo->modulus). x must be coprime with modulus (and thus cannot be 0).
```

Comment:
Isn't 0 coprime with 1 (and no other positive integer)? The gcd is 1, since 1 divides 0 and 1 divides 1.

Reply:
Maybe I'm missing something but this line still says `x coprime with modulus => x != 0`.